custom/icds/location_rationalization/forms.py

```diff
@@ -0,0 +1,46 @@
+from django import forms
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy
+
+from crispy_forms import layout as crispy
+from crispy_forms.bootstrap import StrictButton
+
+from corehq.apps.hqwebapp.crispy import HQFormHelper
+from corehq.util.workbook_json.excel import get_workbook
+
+
+class LocationRationalizationRequestForm(forms.Form):
+    file = forms.FileField(label="", required=True,
+                           help_text=ugettext_lazy("Upload xlsx file"))
+
+    def __init__(self, *args, **kwargs):
+        if 'location_types' in kwargs:
+            self.location_types = kwargs.pop('location_types')
```

Comment:
`kwargs.pop('location_types', None)` would let you drop the if statement.

Reply:
https://github.com/dimagi/commcare-hq/pull/26865/commits/33c80236959e64d07120378b8a9475c16a665579